lms/djangoapps/grades/rest_api/v1/tests/test_gradebook_views.py

```diff
@@ -1461,6 +1461,8 @@ def test_with_override_no_history(self, login_method):
         resp = self.client.get(
             self.get_url(subsection_id=self.usage_key)
         )
+        created_date = resp.data['history'][0]['created']
```

Comment:
You may want to change the name of this test function, since there is in fact a historical record for every override now.  Maybe `test_override_no_modifications`?